Sources/Thumbprint/Components/Toast.swift

```diff
@@ -43,6 +61,14 @@ public class Toast: UIView {
         }
     }
 
+    private static var keyboardTracker: KeyboardTracker?
+
+    /// Call this function inside of UIApplicationDelegate's application(_:didFinishLaunchingWithOptions:)
+    /// to have toasts reposition themselves above the software keyboard
```

Comment:
I don't see this being added to `UIApplicationDelegate`

Reply:
thumbprint-ios is a framework; it doesn't have an app delegate.  Making the call to `registerForKeyboardNotifications()` would have to be done inside of the app that is importing the framework.

Reply:
Ahh, I was under the impression this was iOS repo.